data/ansible/nlu.yml

```
@@ -0,0 +1,10 @@
+version: "3.1"
+
+nlu:
+  - intent: intent_ansible_namespaces_access_issues
+    examples: |
+      - unable to access my ansible namespace
+      - unable to get to my ansible namespace
+      - i used to be able to access my ansible namespace but i can't anymore
+      - are you able to get me access to a ansible namespace
+      - it says i don't have access to my ansible namespace
```

```suggestion
      - it says i don't have access to my ansible namespace
      - ansible namespace access
      - ansible namespace issues
      - ansible namespace problems
      - ansible namespace doesn't work
      - how do I get access to an ansible namespace
      - fix ansible namespace 
```

Our bot is dumb, some more examples